command/CommandController.js

```diff
@@ -573,10 +578,27 @@ async function searchForPocketArticle(getBody, searchTerm, extendedData) {
       logger.debug('keysarr = ' + keysArr);
```

Comment:
I spent some time digging in this code and I think that the parameter 'searchTerm' is not really correct here.  From what I'm seeing, we already know the URL at this point and searchTerm shoudl really be url.  At least this is from the skill perspective.  I know this is not part of the patch, but it causes some confusion in the review.  Can you have a look?

Reply:
I think we should leave searchTerm in the searchForPocketArticle method, because the Pocket API takes it as a search term (https://getpocket.com/developer/docs/v3/retrieve), and we could possibly use this feature for something that is not an url.

However the only place where we call this method is in the processArticleRequest method, where we have an url. So our only use case of searchForPocketArticle is for an url search, but it could also be used for other purposes so it doesn't make sense to rename searchTerm to url in searchForPocketArticle in my opinion

Reply:
Yes if we only use this function for url searches, we should rename it to match that. Makes sense since we are using local search (findBestScoringTitle) for term search.

Reply:
Renamed in commit https://github.com/MozScout/scout-ua/pull/135/commits/252cbae550b45fd97930e1ac755ebe539ce3aa90